### ISSUE TITLE
Missing module/folders in Label Only Jupyter Notebook

### ISSUE THREAD

**Issue by anandseshadri1998** [Trusted-AI/adversarial-robustness-toolbox#1480]
I'm trying to run the jupyter notebook label_only_membership_inference.ipynb, but I'm unable to the first cell and third cell. Seems that the models folder is not included in the current repo. Unsure if this is a problem on my end or the repo.

Thanks in advance,
Anand

First cell:
import torch
from torch import nn
import numpy as np
from models.mnist import Net

Third cell:
state_dict = torch.load("models/mnist_cnn.pt")
model = Net()
model.load_state_dict(state_dict)

**Comment by beat-buesser** [comment]
Hi @anandseshadri1998 Thank you very much for using ART! I think you are right, this notebook does currently not provide a model to run, we'll try to add a model in the future. A short term solution for you would be to train a MNIST classification model in PyTorch and use it to run the notebook.
In case you create your own model, please let me know if you would like to contribute it to ART and fix this issue.
